Automattic/jetpack
# Moved endpoint instantiations to files that declare their classes.

Fixes #6629 

#### Changes proposed in this Pull Request:
This is a workaround to avoid the [PHP bug](https://bugs.php.net/bug.php?id=74543) where API requests would cause PHP workers to crash with a segfault. 

#### Testing instructions:
* Get a site running on PHP 7.1 with opcache enabled.
* Install and connect Jetpack.
* Go to wordpress.com/sites, open your site and open the settings tab.
* See that you're not able to edit anything, and there are segfault reports in your error logs.
* Update Jetpack to this PR.
* See that Jetpack now works fine with Calypso settings.


#### Proposed changelog entry for your changes:
* Added a workaround to make Jetpack fully compatible with PHP 7.1.
